website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,239 @@
+
+For large organizations in highly regulated industries, it’s important to keep track of changes in your feature flag system. When your team members are using Unleash, consider that while some users have permission to modify flags, you can still put guardrails in place to approve or deny the changes.
+
+Imagine a developer working at a large banking platform who wants to modify a feature flag controlling a new authentication method. Instead of making an immediate change, the [change request](/reference/change-requests) workflow could require:
+
+1. Initial proposal submission
+2. Security team review
+3. Compliance officer approval
+4. Final sign-off from technical leadership
+
+With Unleash, you can create a change request workflow to reflect these exact requirements.
+And from a security perspective, you'll always want an auditable trail of changes that occur in your feature flag system. We record all changes like these in the event logs. These are the configuration management practices required by compliance frameworks like SOC 2, FedRAMP, and ISO 27001. We’ll explore [event logs](#audit-manual-and-automated-events-in-unleash) more in a later section, but it’s important to note throughout this tutorial since it’s relevant to many features in Unleash that align with security and compliance standards.
+
+> **Note:** Anyone with project access can create a change request; however, only users with specific permissions can approve, apply, or skip them. None of the [predefined roles](/reference/rbac#predefined-roles) have any change request permissions, so you must create [custom project roles](/reference/rbac#predefined-roles).
+
+![This GIF shows how to quickly make changes to your feature flag, request the change, approve it, and apply the changes.](/img/use-case-user-mgmt-cr.gif)
+
+To get the most out of this functionality, we recommend you use change requests for your production environment to require explicit approval from at least one approver on your team. This is known as the [four-eyes principle](https://www.unido.org/overview-member-states-change-management-faq/what-four-eyes-principle). However, you can add up to 10 approvers to a submitted change request if multiple stakeholders are involved.
+
+Let’s say a project owner wants to update a rollout strategy so that a new feature will increase from 50% exposure to 100% of your end user base. The change is approved by other team members, but the project owner will need to hold off applying the update in production until components from another team are ready to be released. Since the project owner’s changes are ready ahead of time, they’ll need to delay the change. In this case, we recommend [scheduling change requests](/reference/change-requests#scheduled-change-requests) to allow teams to plan and queue feature flag modifications with precise timing, which gives you more control and predictability in a release process.
```

Comment:
Line 139 also has a similar example, maybe not so detailed...

Reply:
Thanks! I think that example isn't about scheduling. we'll update the phrasing to distinguish them better.